### PR TITLE
chore: fix s3 validation tools

### PR DESCRIPTION
**Pull request by jgadling**
These changes update our S3 validation tools so that they handle our recent directory/schema changes:
- Support multiple tomograms per run
- Support multiple tiltseries per run
- Follow new directory naming conventions

It also adds support for running data validation jobs in AWS via our `enqueue_runs.py` script. When we run jobs in the cloud, the test results are available at the following URL's (just replace the dataset id) depending on whether you're validating staging or prod data:

- https://files.cryoet.staging.si.czi.technology/prod_validation/10168/index.html
- https://files.cryoet.staging.si.czi.technology/staging_validation/10168/index.html

These allure reports *do* include history, for up to the last 20 runs.

This PR is supposed to bring our test coverage back up to parity with where it was at before we broke them by our multiple-tomogram-support migrations. It does *not* attempt to reassess which tests are still aligned to our needs.

### CODE REVIEW

File: ingestion_tools/scripts/data_validation/conftest.py
Comment (jgadling) [inline ingestion_tools/scripts/data_validation/conftest.py:101]: We had some tests that assumed one tiltseries per run. This method lets us update those tests to support multiple tiltseries.

File: ingestion_tools/scripts/data_validation/conftest.py
Comment (jgadling) [inline ingestion_tools/scripts/data_validation/conftest.py:130]: We had some tests that assumed a single tomogram per run. This lets us update them to support multiple tomos per run.

File: ingestion_tools/scripts/data_validation/fixtures/data.py
Comment (jgadling) [inline ingestion_tools/scripts/data_validation/fixtures/data.py:176]: mdocs are now in the `Frames` directory, rather than associated with tiltseries

File: ingestion_tools/scripts/data_validation/fixtures/data.py
Comment (jgadling) [inline ingestion_tools/scripts/data_validation/fixtures/data.py:222]: Get all tomogram metadatas for this run+voxelspacing

File: ingestion_tools/scripts/data_validation/fixtures/data.py
Comment (jgadling) [inline ingestion_tools/scripts/data_validation/fixtures/data.py:237]: Support multiple ng configs per voxelspacing

File: ingestion_tools/scripts/data_validation/tests/test_alignments.py
Comment (uermel) [inline ingestion_tools/scripts/data_validation/tests/test_alignments.py:12]: We should also validate the AreTomo alignments, which don't necessarily have a `tlt` file. I made [this package](https://github.com/uermel/cryoet-alignment), which we also use for ingestion, so we should also re-use it here. What I would suggest (to not block this PR) is that we consider updating the alignment validations as a separate scope of work, which I can take on, because there are a lot of technical details to be considered.

File: ingestion_tools/scripts/data_validation/fixtures/data.py
Comment (uermel) [reply ingestion_tools/scripts/data_validation/fixtures/data.py:260]: I think there are things we can validate within the configs themselves, but this is also outside of the scope of this PR I believe.

File: ingestion_tools/docs/enqueue_runs.md
Comment (uermel) [inline ingestion_tools/docs/enqueue_runs.md:161]: This is awesome, but I'm getting an Access Denied message for the staging link. Prod link works.

Comment (uermel) [reply ingestion_tools/docs/enqueue_runs.md:161]: image

**Review by uermel — CHANGES_REQUESTED** [review]
This looks great, some changes to the alignment validation, but I think this is maybe something I should take a look at after this is merged?

File: ingestion_tools/docs/enqueue_runs.md
Comment (jgadling) [reply ingestion_tools/docs/enqueue_runs.md:161]: Yep, I haven't run the validation on every dataset .... YET

Comment (uermel) [reply ingestion_tools/docs/enqueue_runs.md:161]: Sorry, no rush, I didn't realize that was the reason 😅

File: ingestion_tools/scripts/enqueue_runs.py
Comment (manasaV3) [inline ingestion_tools/scripts/enqueue_runs.py:674]: 🔥

File: ingestion_tools/scripts/data_validation/tests/test_alignments.py
Comment (manasaV3) [reply ingestion_tools/scripts/data_validation/tests/test_alignments.py:12]: One suggestion I would offer is to avoid using the same package for generating the alignment values in the validation process. Instead, I recommend explicitly defining it for validation (even though it might be a little redundant), as this would make it easier to identify any potential regressions.

File: ingestion_tools/scripts/data_validation/tests/test_frame.py
Comment (manasaV3) [inline ingestion_tools/scripts/data_validation/tests/test_frame.py:62]: I believe this is an existing test, but is it a valid case, when there are lesser number of entries in the mdoc than the actual number of frames?

File: ingestion_tools/scripts/data_validation/tests/test_frame.py
Comment (manasaV3) [inline ingestion_tools/scripts/data_validation/tests/test_frame.py:66]: Should this be a blocking test or should this be categorized as a warning?

File: ingestion_tools/scripts/data_validation/tests/test_alignments.py
Comment (uermel) [reply ingestion_tools/scripts/data_validation/tests/test_alignments.py:12]: I agree, it's just a lot of parsing code to rewrite because the ALN files unfortunately are not simple column-based files. Totally doable though. FWIW, the parser I wrote has a bunch of tests, so maybe we are more interested in making sure those are good and trust the parser?

File: ingestion_tools/scripts/data_validation/tests/test_alignments.py
Comment (jgadling) [reply ingestion_tools/scripts/data_validation/tests/test_alignments.py:57]: Should we remove this test?

File: ingestion_tools/scripts/data_validation/tests/test_frame.py
Comment (uermel) [reply ingestion_tools/scripts/data_validation/tests/test_frame.py:66]: This should be a blocking validation. We should have a frame file for every mdoc section.

File: ingestion_tools/scripts/data_validation/tests/test_alignments.py
Comment (uermel) [reply ingestion_tools/scripts/data_validation/tests/test_alignments.py:57]: I think this is because before we were not distinguishing `rawtlt` and `tlt` files. It is fine to do this check for `rawtlt`, just not for `tlt`.

**Review by uermel — APPROVED** [review]

**Comment by uermel** [comment]
From offline conversation with @jgadling: merge this and deal with fixing alignment validations later.

**Pull request merged**
